swagger.yaml

```diff
@@ -4214,6 +4237,19 @@ definitions:
       - 'all'
       - ''
 
+  operator-type:
+    type: string
+    enum:
+      - 'lso'
+      - 'ocs'
+
+  list-operators:
```

Comment:
You are allowing users to set the status/status_info in create/update cluster

Reply:
fixed